tests/integration-tests/configManager/configHandler-test.js

```diff
@@ -34,7 +34,7 @@ const updateBodyWithTypes = {
     },
     prometheus_metrics: {
         push_gateway_url: 'string_value',
-        buckets_sizes: 'string_value',
+        buckets_sizes: [2.1, 1, 10, 17, 19, 0.0],
```

Comment:
I suggest change the examples to:
[0.01, 0.05, 0.10, 0.25, 0.50, 0.100, 0.200, 0.300, 0.400, 0.500, 1, 2, 5, 10, 30, 60, 120]

Reply:
Sure, changed now.